src/lib/components/Input.svelte

```diff
@@ -1,24 +1,21 @@
 <script lang="ts">
+  // props for the input
   export let type = 'text';
   export let name: string;
-  export let label: string;
   export let placeholder: string | undefined = undefined;
   export let required = false;
   export let id = '';
   export let value = '';
-  export let autocomplete = false;
+  export let autocomplete = '';
+  export let inputClass = 'mt-1 focus:ring-indigo-500 focus:border-indigo-500 block w-full shadow-sm sm:text-sm border-gray-300 rounded-md';
+
+  // props for the label
+  export let label = '';
+  export let labelClass = 'block text-sm font-medium text-gray-700';
+  export let labelFor = '';
```

Comment:
Label's `for` and Input's `name` share the same value, so there's no need for `labelFor`.
The class for the label should not change, its part of the component definition.

```suggestion
export let label: string | undefined = undefined;
```